x-pack/plugins/cases/public/components/visualizations/actions/open_modal.test.tsx

```diff
@@ -71,10 +89,12 @@ describe('openModal', () => {
         },
       },
     });
+
     (toMountPoint as jest.Mock).mockImplementation((node) => {
       ReactDOM.render(node, element);
       return mockMount;
     });
+
     jest.clearAllMocks();
     openModal(mockLensApi, 'myAppId', {} as unknown as CasesActionContextProps, getMockServices());
```

Comment:
These tests are very weird. `openModal` is called on `beforeEach`??

Anyway, I wanted to suggest a test where the `timerange` in `mockLensApi` is absolute(right now we only test relative to absolute).

Reply:
Great point! I added the extra tests in  [`93ed7f2` (#189168)](https://github.com/elastic/kibana/pull/189168/commits/93ed7f205482489a54c33869deb46fb76fe78ffd)